helpers/tf_cfg.py

```diff
@@ -158,6 +196,81 @@ def check(self):
             msg = 'running clients on a remote host "%s" is not supported' % client_hostname
             raise ConfigError(msg)
 
+    def configure_logger(self):
+        """Configure a logger."""
+        pretty.install()
+
+        date_format = "%y-%m-%d %H:%M:%S"
+        file_handler = logging.FileHandler(self.get("General", "log_file"))
+        file_handler.setFormatter(
+            logging.Formatter(
+                fmt="%(asctime)s.%(msecs)03d | %(levelname)s | %(message)s", datefmt=date_format
+            )
+        )
+        stream_handler = RichHandler()
+        stream_handler.setFormatter(
+            logging.Formatter(fmt=" | %(message)s", datefmt=date_format + ".%f")
+        )
+        self.logger.addHandler(file_handler)
+        self.logger.addHandler(stream_handler)
+
+        # set default logging level as in config file,
+        # otherwise a correct log level sets up during initialisation of cmd arguments after config is initialised.
+        # default level for logging module is Warning
+        self.logger.setLevel(
+            bring_log_level(
+                self.get("General", "verbose"),
+            ),
+        )
+
+    @property
+    def net_devices(self) -> str:
```

Comment:
I don't think that adding a property for a single configuration parameter is a good idea. We should follow the old way. You can get this property using `cfg.get("Tempesta", "interfaces")`. Or you must change the logic for all configurations... but it will take a lot of time.

I propose to remove 58, 70, 227-273 lines

Reply:
It has a reason. It is created not to an easy access to an argument and I understand I can get it via cfg.get("Tempesta", "interfaces"), networking devices are mentioned in several places around the config, and I thought all devices should be configured during Tempesta init process. So, I implemented it as is (using the approach allows us to invoke calculation process for the parameter only one time in the beginning). If we do no need to worry about devices from `"Server", "aliases_interface"` I will update my code.